audio-worklet/design-pattern/awn/index.html

```diff
@@ -80,7 +83,7 @@ <h1 id="title-header"></h1>
           function. See the article on the AudioWorklet design pattern for more
           details.`,
       htmlSource: 'audio-worklet/design-pattern/awn/',
-      demoFunction: runDemo
+      demoFunction: runDemo,
```

Comment:
Did eslinter actually add the trailing comma?  It's not really wrong not to have it, is it?

Reply:
Yes. Surprisingly this is recommended/enforced by Google ESlint plugin. I kinda like it because the change can be simpler after this treatment.